package.json

```diff
@@ -21,15 +21,15 @@
     "express": "4.15.2",
     "fh-mbaas-api": "6.1.8",
     "fh-wfm-appform": "0.1.0",
-    "fh-wfm-file": "0.1.1",
-    "fh-wfm-mediator": "0.3.1",
-    "fh-wfm-message": "0.1.1",
-    "fh-wfm-result": "0.2.1",
+    "fh-wfm-file": "0.1.2",
+    "fh-wfm-mediator": "0.3.2",
+    "fh-wfm-message": "0.1.2",
+    "fh-wfm-result": "0.2.2",
     "fh-wfm-simple-store": "0.2.1",
```

Comment:
I think this should be `fh-wfm-simple-store@0.2.2` from feedhenry-raincatcher/raincatcher-simple-store#18